sense/dashboard/values.yaml

```
@@ -45,11 +45,11 @@ dashboard:
   # CPU and memory limits for the dashboard app
   resources:
     requests:
-      memory: '2Gi'
+      memory: '1Gi'
```

This is still more than it really needs.  
```suggestion
      memory: 256Mi'
```